ingestion/src/metadata/orm_profiler/processor/orm_profiler.py

```diff
@@ -124,9 +132,44 @@ def get_table_profile_sample(self, table: Table) -> Optional[float]:
             if my_record_tests and my_record_tests.profile_sample:
                 return my_record_tests.profile_sample
 
-        return table.profileSample
+        return table.profileSample or 100.0
+
+    def get_partition_details(
+        self,
+        orm: DeclarativeMeta,
+        table: Table,
+        partition_details: Optional[Dict] = None,
+    ) -> Optional[Dict]:
+        """Get partition details for the profiler if working with
+        bigquery table
+
+        Args:
+            table: table entity
+        Returns:
+            None or Dict
+        """
 
-    def build_profiler(self, orm: DeclarativeMeta, table: Table) -> Profiler:
+        if table.serviceType == DatabaseServiceType.BigQuery:
```

Comment:
did we open a ticket regarding this topic? Ideally all partition information should come from the ingestion so that here we can directly reference table.partitionDetails.

When opening the ticket, we could reference it here

Reply:
Correct, we have not open a ticket regarding this, will do right now. You are referring to this part (https://github.com/open-metadata/OpenMetadata/pull/5441#discussion_r896393247) not picking up partition logics, right?